### DISCUSSION TITLE
news/2022/07/more-image-processing-methods/

### DISCUSSION THREAD

**Discussion by giscus** [razonyang/hugo-theme-bootstrap#915] [General]
# news/2022/07/more-image-processing-methods/

Since v1.0.0-alpha.1, HBS supports much more image processing methods. Such as Crop, Fit and Fill images. You can also apply filters on an image.

https://labs.embed.co.ke/news/2022/07/more-image-processing-methods/

**Comment by k-kipruto** [comment]
Hello world

**Comment by k-kipruto** [comment]
Hello world

**Reply by razonyang** [reply]
Hi, to get notified when.users commentted, you may need to change the giscus and/or utterance parameter with your own.
